pricing-page-exercise/onder/index.html

```diff
@@ -4,10 +4,114 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./styles.css">
+<link rel="stylesheet" href="./styles.css">
   <title>Pricing page Exercise</title>
 </head>
 <body>
-  
+ <header>
+    <nav class="navbar">
+        <div><h3 class="heading-small">Company name</h3></div>
```

Comment:
If this would be a real website, usual behaviour is when a user clicks on the logo/company name in the navbar, he/she is redirected to the main page. So using a link element (<a></a>) instead of <h3></h3> would be more appropriate.